recipes/ledger/meta.yaml

```diff
@@ -0,0 +1,63 @@
+{% set name = "ledger" %}
+{% set version = "3.1.3dev16" %}
```

Comment:
We generally don't build dev packages due to CI limits.  Is there a full release that is available?

Reply:
@scopatz The latest full release doesn't build with conda-forge's pinned boost versions. I was hoping to have this dev build until I ask the devs to create a new release with my fixes (https://github.com/ledger/ledger/pull/1823) to build with boost 1.70.

Reply:
To be clear, I would update this pull request once I get the fixes merged to build a new dev version and then try to get a new release created that can be built with conda-forge. Let me know if you'd rather leave this pull request open until the full release is available, but I'm unsure on how long this will take.

Reply:
I think I'd prefer to wait, but someone else on @conda-forge/staged-recipes might be more open to it.

Reply:
@twrightsman, you can build the release but add your PR as a patch. See https://conda.io/projects/conda-build/en/latest/resources/define-metadata.html#patches

Reply:
@isuruf That looks like a great solution, I'll give it a shot

Reply:
@twrightsman, can you only use the diff from ledger/ledger#1823 and remove other changes in the dev17.patch?

Reply:
Okay, I'll add another commit

Reply:
@isuruf dev17.patch may contain fixes besides mine since v3.1.3 that are required for proper building, but we can try it

Reply:
I'll need to cherry-pick the master commits to avoid feature changes and include any required build changes, will update the patch file when I find a minimal working set